CMakeLists.txt

```diff
@@ -294,6 +295,42 @@ else()
     message( STATUS "${red}No HIP/ROCm support: gpu_backend = ${gpu_backend}${plain}" )
 endif()
 
+#-------------------------------------------------------------------------------
+# OneMKL/SYCL support.
+message( "" )
+set( blaspp_use_onemkl false )  # output in blasppConfig.cmake.in
+# if cuda or hip were found gpu_backend was set appropriately
+if (gpu_backend MATCHES "^(onemkl|auto)$")
+
+    message( STATUS "${bold}Looking for OneMKL${not_bold} (gpu_backend = ${gpu_backend})" )
```

Comment:
I don't know why, but Intel uses lowercase "one" everywhere: oneAPI, oneMKL.
Cf. https://www.intel.com/content/www/us/en/docs/oneapi/programming-guide/2023-0/intel-oneapi-math-kernel-library-onemkl.html

Reply:
I have replaced OneMKL with oneMKL in blaspp.